projects/js-packages/components/components/record-meter-donut/index.tsx

```diff
@@ -40,21 +54,29 @@ const DonutMeter: React.FC< DonutMeterProps > = ( {
 	backgroundColor = '#00BA37', // jetpack green fallback
 	thickness = '3.5',
 	donutWidth = '64px',
+	title = '',
+	description = '',
 } ) => {
 	const count = () => {
 		// get count as a percent value
 		return ( segmentCount / totalCount ) * 100;
 	};
 
+	// If we don't have a title or description, hide the meter from screen readers.
+	const isHidden = title.length === 0 && description.length === 0 ? 'true' : 'false';
```

Comment:
This line would cause a render-breaking error if `title` or `description` were set to `null`. I assume the TypeScript tooling should prevent such usage, but it might be good to add some defensive logic here like so:

```suggestion
	const isHidden = typeof title === 'string' && title.length === 0 && typeof description == 'string' && description.length === 0 ? 'true' : 'false';
```

Reply:
Thanks for pointing this out, Jason. I've addressed this in the incoming commits.